### PR TITLE
added 'add_custom_facts_from_yaml' function to load default facts from YAML file in <module>/spec

### PR DESCRIPTION
**Pull request by exodusftw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -288,6 +288,19 @@ You can also globally set facts in `spec/spec_helper.rb`
     add_custom_fact :root_home, ->(_os,facts) { "/tmp/#{facts['hostname']}" }
     ```
 
+  * Load facts directly from YAML file in <module>/spec and confined/excluded as above
+
+     ```ruby
+      # Assign fact import to variable as a constant
+      # Ensure File is in <module>/spec dir and is prefaced with 'spec' i.e. 'spec/<file>'
+      GLOBAL_FACTS = add_custom_facts_from_yaml 'spec/<yml_file>'
```
Comment (rnelson0) [inline README.md]: Do you need to assign it to a variable?

Comment (rnelson0) [inline README.md]: 294-296 have an odd indent.

**Comment by logicminds** [comment]
This is probably no longer needed.  I added support for custom facts in facterdb. 

https://github.com/camptocamp/facterdb#supplying-custom-external-facts

You will need to update your facterdb gem and following the instructions in the url above.  Basically just set a environment variable.

**Comment by rodjek** [comment]
Going to close this out due to age and that the functionality has been added to FacterDB as @logicminds mentioned.

**Pull request closed without merging**